frontstage/templates/surveys/surveys.html

```diff
@@ -49,7 +49,7 @@
             <span id="status-{{ loop.index }}" name="status">{{ survey.status }}</span>
         </div>
         <div class="grid__col col-2@m">
-            {% if not survey.status == 'Complete' or not survey.collection_instrument_type == 'EQ' %}
+            {% if not survey.display_access_button %}
```

Comment:
This is currently only showing the access link if it *shouldn't* be displayed.

Probably worth changing to:
```python
{% if not survey.is_closed %}
...
```
to avoid confusion.

Reply:
Spoke to @shahi645 and makes sense to keep `.display_button` but should be changed to `{% if survey.display... %}`